internal/watermill/driver/kafka/metrics/adapter.go

```diff
@@ -0,0 +1,202 @@
+package metrics
```

Comment:
@sagikazarmark this is not pretty, but I couldn't find an opensource bridge/adapter supporting this. We can consider putting this into a separate opensource package with an extended review, WDYT?

Reply:
Decision we will publish this as a separate package (I will spend some time creating the repo this week or the week after)